config.toml.example

```diff
@@ -1,9 +1,21 @@
 [mastodon]
 server = "https://tomkahe.com"
 access_token = ""
+screenshot_status = "#pokemon #gameboy #nintendo #FediPlaysPokemon"
```

Comment:
I haven't gotten a chance to test this PR yet, but will this let me plug in the previous action at all, like:

screenshot_status = "$PREV_ACTION\n\n#pokemon #gameboy #nintendo #FediPlaysPokemon"